src/test/java/core/basesyntax/service/FileReaderImplTest.java

```diff
@@ -0,0 +1,56 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class FileReaderImplTest {
+    private static final String FILES_ROOT = "src/test/java/resources/";
+    private static FileReader fileReader;
+
+    @BeforeAll
+    static void beforeAll() {
+        fileReader = new core.basesyntax.serviceimpl.FileReaderImpl();
+    }
+
+    @Test
+    void readLines_validFile_ok() {
+        List<String> expected = List.of("b,apple,100",
+                "b,grapes,50",
+                "p,apple,50",
+                "r,apple,20",
+                "p,grapes,25",
+                "s,pears,40");
+        List<String> actual = fileReader.readFromFile(FILES_ROOT + "testFileOk.csv");

```

Comment:
it's better to create several different constants for file pathes instead of concatenate new String object